src/components/Carousel.tsx

```diff
@@ -1,22 +1,30 @@
 import React, { useEffect, useState } from 'react';
 
 import './styles/Carousel.scss';
-
 import NextSvg from '../assets/next_btn.svg';
 import PrevSvg from '../assets/prev_btn.svg';
 
+/* eslint-disable @typescript-eslint/no-empty-function */
+export const ChangeSlideContext = React.createContext({next: () => { }, prev: () => { }});
```

Comment:
remove the eslinter ignore line for the following

```suggestion
export const CarouselContext = React.createContext({
  next: () => undefined,
  prev: () => undefined,
});
```